lib/bgs/services/corporate_update.rb

```diff
@@ -12,11 +12,15 @@ def self.service_name
       'corporate_update'
     end
 
+    def bean_name
+      'CorporateUpdateServiceBean'
+    end
+
     # update a POA relationship
     def update_poa_access(participant_id:, poa_code:, allow_poa_access: 'y', allow_poa_c_add: 'y')
       response = request(
         :update_poa_access,
-        "updatePoaAccess": {
+        {
```

Comment:
these are a bit odd to me, its almost like someone implemented this but never tested or used it...? 🤷🏻